client.go

```diff
@@ -111,6 +111,10 @@ type ClientConfig struct {
 	// MaxLSATFee is the maximum that we are willing to pay in routing fees
 	// to obtain the token.
 	MaxLSATFee btcutil.Amount
+
+	// LoopOutMaxParts defines that maximum number of part that may be used
```

Comment:
nit: "that maximum number of part" => "the maximum number of parts"

Reply:
Maybe it'd be also good to indicate that "parts" refer to multipart payment.

Reply:
clarified